### PR TITLE
Basic bot

### PR DESCRIPTION
**Pull request by ef4203**
Connection to discord api

### CODE REVIEW

File: bot/core/context.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        ------
+        commands.CommandError
+            Will be raised when posting to hastebin fails.
+        """
+
+        headers = {
+            'Content-Type': content_type or 'text/plain',
+        }
+
+        async with self.session.post('https://hastebin.com/documents', data=data.encode('utf-8'), headers=headers) as document:
```
Comment (ef4203) [inline bot/core/context.py]: This feature is not on the project board, please remove and create an item on the project board for it.

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+git+https://github.com/Rapptz/discord.py@rewrite#egg=discord.py
+psutil
```
Comment (ef4203) [inline requirements.txt]: What is this? Add a comment for each dependency explaining what it is and why you need it.

File: bot/core/bot.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+import aiohttp
+import psutil
+from discord.ext import commands
+
+VersionInfo = collections.namedtuple('VersionInfo', 'major minor micro releaselevel serial')
+
+
+class Bot(commands.Bot):
+    __version__ = '0.0.1a'
```
Comment (ef4203) [inline bot/core/bot.py]: Remove hard coded version, create an item on the project board 'Version info' or something.

File: requirements.txt (hunk 2 above)
Comment (itsVale) [reply requirements.txt]: Done.

**Comment by ef4203** [comment]
@itsVale

File: bot/core/bot.py (hunk 3 above)
Comment (itsVale) [reply bot/core/bot.py]: Removed everything related to the bot's version.

File: bot/core/context.py (hunk 1 above)
Comment (itsVale) [reply bot/core/context.py]: Added an entry for the bot's command context to the project board.

**Pull request merged**